auth-method-oauth2.html

```diff
@@ -234,6 +234,7 @@
       max-width: 560px;
       width: calc(100% - 16px);
       box-sizing: border-box;
+      @apply(--auth-redirect-section);
```

Comment:
You've added it to 3 different selectors. I'm guessing you either meant that and then the name is misleading or you wanted to apply it to the redirect section only. 
Either way you have to change it.